### PR TITLE
DOCS-466: Query profiler settings docs

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation

Documented #6283

### CODE REVIEW

File: docs/en/operations/settings/settings.md [hunk 1]
```diff
@@ -994,4 +994,52 @@ Lower values mean higher priority. Threads with low `nice` priority values are e
...
+
+Sets the period for a real clock timer of the query profiler. Real clock timer counts wall-clock time.
+
+Possible values:
+
+- Positive integer number of nanoseconds.
+
+    Recommended values:
+        
+        - 10000000 (100 times a second) nanosecods and more for for single queries.
```
Comment (blinkov) [inline docs/en/operations/settings/settings.md]:
```suggestion
        - 10000000 (100 times a second) nanoseconds and less for single queries.
```

File: docs/en/operations/settings/settings.md [hunk 2]
```diff
@@ -994,4 +994,52 @@ Lower values mean higher priority. Threads with low `nice` priority values are e
...
 Default value: 0.
 
+
+## query_profiler_real_time_period_ns {#query_profiler_real_time_period_ns}
+
+Sets the period for a real clock timer of the query profiler. Real clock timer counts wall-clock time.
+
+Possible values:
+
+- Positive integer number of nanoseconds.
```
Comment (blinkov) [inline docs/en/operations/settings/settings.md]:
```suggestion
- Positive integer number, in nanoseconds.
```

File: docs/en/operations/settings/settings.md [hunk 3]
```diff
@@ -994,4 +994,52 @@ Lower values mean higher priority. Threads with low `nice` priority values are e
...
+    Recommended values:
+        
+        - 10000000 (100 times a second) nanosecods and more for for single queries.
+        - 1000000000 (once a second) for cluster-wide profiling.
+
+- 0 for turning off the timer.
+
+Type: [UInt64](../../data_types/int_uint.md).
+
+Default value: 1000000000 nanoseconds.
```
Comment (blinkov) [inline docs/en/operations/settings/settings.md]:
```suggestion
Default value: 1000000000 nanoseconds (once a second).
```

File: docs/en/operations/settings/settings.md [hunk 4]
```diff
@@ -994,4 +994,52 @@ Lower values mean higher priority. Threads with low `nice` priority values are e
...
+
+**See Also**
+
+- [system.trace_log](../system_tables.md#system_tables-trace_log)
+
+## query_profiler_cpu_time_period_ns {#query_profiler_cpu_time_period_ns}
+
+Sets the period for a CPU clock timer of the query profiler. Real clock timer counts CPU time.
```
Comment (blinkov) [inline docs/en/operations/settings/settings.md]:
```suggestion
Sets the period for a CPU clock timer of the query profiler. This timer counts only CPU time.
```

**Review by blinkov — CHANGES_REQUESTED** [review]

**Comment by nikitamikhaylov** [comment]
No need to change submodule in documentation PR.

**Pull request merged**